### ISSUE TITLE
[Accessibility]: Code Insights Edit UI - Repositories input not read correctly

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41685]
Note: This issue should not be worked on until after https://github.com/sourcegraph/sourcegraph/issues/41684 is resolved.

### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33501

### Problem description

Voice over reads this when page first loads "Repositories loading Separate repositories with commas example github.com/sourcegraph/sourcegraph"

Problems

- The "Repositories" I believe is for the label. It should announce that this is a label for an input
- "loading" is coming from the chart preview loading. I think we should suppress the loading announcement for this chart from screen readers. It reloads _a lot_ and can be overwhelming while editing data series input.

### Expected behavior

- Repositories label should announce itself as a label belonging to an input
- Loading should be suppressed so that it doesn't interrupt explanations of other items on the screen

### Additional details

Related "loading" issue https://github.com/sourcegraph/sourcegraph/issues/41684

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @joelkw @felixfbecker @vovakulikov

**Comment by felixfbecker** [comment]
We probably should remove the `autofocus` from this field. Then the user would navigate top-to-bottom and encounter the label first, instead of being teleported right into the input with no orientation.

**Comment by felixfbecker** [comment]
But there also seems to be an issue here with the "Repositories" label not being properly associated. Looking at the DOM, there are **multiple** _nested_ `<label>`s, which is most definitely wrong:



![Image](https://user-images.githubusercontent.com/10532611/193698836-417457f1-694c-4d10-b6f9-3041099aebab.png)

**Comment by felixfbecker** [comment]
However, I would suggest we _don't_ look at this issue **this iteration (28)** because we don't even know if in a month we will even still have this input field. After this iteration, we should have more clarity on that (this iteration is settling on a solution and design for strat scoped insights).

**Comment by vovakulikov** [comment]
@felixfbecker This problem with nested labels comes from the wildcard textarea implementation, it always renders textarea with wrapped label element. 

I actually don't know is it our scope or not (probably our problem since front-end platform team doesn't exist anymore)

We should adjust Textarea component and don't render label always (but do this conditionally and only if label prop is provided) 

I think I may agree with autofocus comment above, but I'm not 100% sure because I personally found this useful, and also this problem with autofocus seems like a generic problem that not only creation UI pages have. We have autofocus usage all over the web app client. 

I suggest we address points about nested labels first in this issue and address autofocus attribute problem in a separate issue.
